addon/components/nypr-o-header.js

```diff
@@ -136,6 +136,10 @@ export default Component.extend({
     document.body.classList.toggle('side-menu-is-active', force);
 
     this.setMenuHeightOffset();
+
+    if (this.get('isOpen') === true){
```

Comment:
in newer versions of ember, you don't need to call `.get()` on simple bound attributes. you can access it with simply `this.isOpen`